update deh lumps.

Voros2 opened the pull request.

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -73,33 +73,33 @@ par 32 0
 STARTUP5============================================================================\nThis is FreeDM, the free content deathmatch FPS.\n\nFreeDM is freely redistributable under the terms of the modified BSD\nlicense. Check out the Freedoom website for more information:\n\n    https://freedoom.github.io/\n============================================================================
-GOTARMOR = picked up a light armor vest.
-GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
```
Keeping these describe the actual effect (1%) rather than something vague I think is appropriate considering it's a deathmatch WAD. I wanted to do the same for the other health bonuses as well, but it's not possible since they depend on the skill level.

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -73,33 +73,33 @@ par 32 0
-GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
-GOTARMBONUS = got a 1% armor bonus.
+GOTARMOR = picked up a lite energy vest.
```
I think calling it an "energy vest" is confusing since it can be confused with the cells ammo type ("polaric energy"). Best to just keep it as is I think?

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -73,33 +73,33 @@ par 32 0
+GOTINVIS = you got the stealth drive!
 GOTSUIT = picked up a hazard suit.
-GOTMAP = got the tactical survey map.
+GOTMAP = got the architecture device.
```
Don't like this - it's very vague.

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -73,33 +73,33 @@ par 32 0
+GOTMAP = got the architecture device.
 GOTVISOR = got the night vision goggles.
-GOTMSPHERE = you got the ultra-overdrive sphere!
+GOTMSPHERE = you got the ultimate drive!
```
"drive" is vague here. Find another term.

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -109,7 +109,7 @@ GOTCHAINSAW = you got the angle grinder!
 GOTPLASMA = you got the polaric energy cannon!
 GOTSHOTGUN = you got the pump-action shotgun!
-GOTSHOTGUN2 = you got the double-barrelled shotgun!
+GOTSHOTGUN2 = you got the double shotgun!
```
Double what? If it's double barrelled, why not keep it as is?

fragglet commented:
I see that some of these changes match recent changes to `dehacked.lmp`. I think some of these are confusing and should probably be changed back.

Voros2 commented:
Better now? Changed the armor lines back, but kept the bonuses lines (it really won't matter, as players rely on visual clues rather than some small text). Reverted to double-barrelled, tactical survey map. Used "totem" instead "drive" (since "rune" was taken by Doom 2016, and "tablet" sounds weird).

Proluiz commented:
since you talking about dehacked, i have noticed of doom shotgun obtuary is used instead freedoom one, you can do a chack in it?

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -75,31 +75,31 @@ par 32 0
-GOTINVIS = you got the stealth sphere!
+GOTINVUL = you are untouchable!
+GOTBERSERK = punch 'em to death!
+GOTINVIS = you got the stealth totem!
```
Why "totem"? Makes me think of a totem pole. I don't think this works in English.

fragglet commented on lumps/fdm_deh.lmp:
```diff
@@ -75,31 +75,31 @@ par 32 0
 GOTMEGA = picked up a heavy armor vest.
-GOTHTHBONUS = got a 1% health bonus.
-GOTARMBONUS = got a 1% armor bonus.
+GOTHTHBONUS = got a vitality boost.
```
I'm going to have to push back on this. Is there a reason to use "vitality" instead of "health" or "shield" instead of "armor"? This is terminology common across almost all FPS games (and non-FPS games too). At the very least we should be consistent - why should it be called "health" on the status bar but "vitality" in the messages? It's needlessly confusing and I don't understand why you want to change it.

Voros2 commented:
How about "core"?

Voros2 commented:
I don't see the problem with "shield" and "armor". Hacx did it, no one complained. It's only cosmetic. Plus it makes it more different and unique from Doom, which basically uses "armor shards" or "armor bonuses".

chungy commented:
@Voros2 These names are fairly generic and common in the genre. We don't need to be different from Doom here just to be different

fragglet commented:
Right - what @chungy said. These are standard terms used across pretty much all FPS games (and plenty of non-FPS games too). If I was encountering this the first time I'd be wondering "why did they call it shield instead of health?" - it's needlessly confusing.

Voros2 commented:
It's still "health" and "armor". Can the "boost" and "charge" stay?
